### PR TITLE
 Add a hook for ``mimesis``, which has a data files.

### PR DESCRIPTION
**Pull request by arossert**

### CODE REVIEW

File: news/365.new.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add a hook for 'mimesis', which has a data files.
```
Comment (rokm) [inline news/365.new.rst:1]: `'mimesis'` -> ``` ``mimesis`` ```

File: src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-mimesis.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
```
Comment (rokm) [inline src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-mimesis.py:2]: 2022 (even though it does sometimes feel like it's still 2020...).

Comment (arossert) [reply src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-mimesis.py]: Done

File: news/365.new.rst (hunk 1 above)
Comment (arossert) [reply news/365.new.rst]: Done

**Review by rokm — APPROVED** [review]

**Comment by arossert** [comment]
@bwoodsend @rokm thanks for merging this so quickly. 
I wanted to ask what is the “release” policy of this repo? When can I expect this change to be in an official release?
I can see that the latest release was at Nov 29, 2021 (2021.4).

**Comment by bwoodsend** [comment]
It's supposed to be monthly but we're never organised enough to remember. I've released it now.

**Pull request merged**